vignettes/parallel.Rmd

```diff
@@ -58,3 +58,96 @@ There is some overhead associated with running tests in parallel:
 This overhead generally means that if you have many test files that take a short amount of time, you're unlikely to see a huge benefit by using parallel tests.
 For example, testthat itself takes about 10s to run tests in serial, and 8s to run the tests in parallel.
 That's because most tests take less than a second to run, and the slowest test (which takes 3s) is started last.
+
+## Reporters
+
+### Default reporters
+
+See `default_reporter()` for how testthat selects the default reporter.
+In short, testthat selects `ProgressReporter` for non-parallel and `ParallelProgressReporter` for parallel tests by default.
```

Comment:
Is this true for `R CMD check` as well? (Is the CheckReporter no longer used?)

Reply:
No, you are right, `test_check()` uses `CheckReporter` by default. Let me rephrase this.